cli/src/pcluster/config/cluster_config.py

```diff
@@ -2192,7 +2200,48 @@ def _register_validators(self):
             )
 
 
-class SchedulerPluginClusterConfig(BaseClusterConfig):
+class CommonSlurmAndSchedulerPluginClusterConfig(BaseClusterConfig):
```

Comment:
I think this name is not a good choice because we're adding the information about the classes that are inheriting it.

We can call it `CommonSchedulerClusterConfig` or `OnPremSchedulerClusterConfig` (if you want to highlight this is not for Batch) or something else.

Reply:
Done